workshops/create-custom-supply-chains.yaml

```diff
@@ -0,0 +1,294 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:base64", "base64")
+#@ load("@ytt:json", "json")
+---
+apiVersion: learningcenter.tanzu.vmware.com/v1beta1
+kind: Workshop
+metadata:
+  name: scc-workshop-dev
+spec:
+  title: Supply Chain Choreographer - Development Version
+  description: A workshop that demonstrates the capabilities Supply Chain Choreographer provides and how to customize the supply chains.
+  vendor: tanzu.vmware.com
+  authors:
+    - Timo Salm
+    - Gary Gutman
+    - Deepak Khopade
+  #! The difficulty field indicates the target audiences of the workshop. 
+  #! The value can be: beginner, intermediate, advanced, or extreme.
+  difficulty: intermediate
+  #! The duration field gives the maximum amount of time the workshop takes to complete. 
+  #! This field provides informational value and does not guarantee how long a workshop instance lasts. 
+  #! The field format is an integer number with s, m, or h suffix.
+  duration: 3h
+  #! url: https://github.com/mrgaryg/tap-cartographer-workshop
+  #! url: #@ data.values.workshop_git.git_url + "/" + data.values.workshop_git.git_org_name + "/" + data.values.workshop_git.git_repo
+  url: https://github.com/dkhopade/tap-cartographer-workshop
+  content:
+    image: #@ "{}/tap-workshop/cartographer-workshop:latest".format(data.values.container_registry.hostname)
+    files: #@ "{}/{}/tap-cartographer-workshop/workshop?ref=se-supplychains-workshop-dev".format(data.values.workshop_git.git_url,data.values.workshop_git.git_org_name)
```

Comment:
We don't have `workshop_git` in the values for the WaaS catalog.  You'd probably want to point this to your source repo for the workshop content you want to pull in (github.com, for example).  Check out some of the other workshop definitions to see some examples.